ts/src/kucoin.ts

```diff
@@ -1027,42 +1043,41 @@ export default class kucoin extends Exchange {
         //        "retry": false,
         //        "data": [
         //            {
-        //                "withdrawMinFee": "0.0005",
-        //                "chainName": "BTC",
-        //                "preDepositTipEnabled": "false",
-        //                "chain": "btc",
+        //                "status": "enabled",
+        //                "currency": "BTC",
         //                "isChainEnabled": "true",
-        //                "withdrawDisabledTip": "",
-        //                "walletPrecision": "8",
+        //                "chain": "btc",
+        //                "chainName": "BTC",
         //                "chainFullName": "Bitcoin",
-        //                "orgAddress": "",
+        //                "walletPrecision": "8",
         //                "isDepositEnabled": "true",
-        //                "withdrawMinSize": "0.001",
-        //                "depositDisabledTip": "",
-        //                "userAddressName": "",
-        //                "txUrl": "https://blockchain.info/tx/{txId}",
-        //                "preWithdrawTipEnabled": "false",
-        //                "withdrawFeeRate": "0",
-        //                "confirmationCount": "2",
-        //                "currency": "BTC",
         //                "depositMinSize": "0.00005",
+        //                "confirmationCount": "2",
         //                "isWithdrawEnabled": "true",
-        //                "preDepositTip": "",
+        //                "withdrawMinSize": "0.001",
+        //                "withdrawMinFee": "0.0005",
+        //                "withdrawFeeRate": "0",
+        //                "depositDisabledTip": "Wallet Maintenance",
+        //                "preDepositTipEnabled": "true",
+        //                "preDepositTip": "Do not transfer from ETH network directly",
+        //                "withdrawDisabledTip": "",
+        //                "preWithdrawTipEnabled": "false",
         //                "preWithdrawTip": "",
-        //                "status": "enabled"
+        //                "orgAddress": "",
+        //                "userAddressName": "Memo",
         //            },
         //        ]
         //    }
         //
         const responses = await Promise.all (promises);
-        const responseCurrencies = responses[0];
-        const responseChains = responses[1];
-        const data = this.safeValue (responseCurrencies, 'data', []);
-        const chainsData = this.safeValue (responseChains, 'data', []);
-        const currencyChains = this.groupBy (chainsData, 'currency');
+        const currenciesResponse = this.safeValue (responses, 0, {});
+        const currenciesData = this.safeValue (currenciesResponse, 'data', []);
```

Comment:
@ttodua is the internal API working for you? getting this 403 error
![image](https://github.com/ccxt/ccxt/assets/43336371/373012c4-ebc5-4258-9620-dfd5e8c0577d)